### ISSUE TITLE
High-res version of your logo

### ISSUE THREAD

**Issue by dankohn** [firehol/netdata#3801]
Could you please share a high-res version of your logo (preferably SVG or AI) so that we can add the project to the monitoring section of the CNCF cloud native interactive landscape: 

https://landscape.cncf.io/grouping=landscape&landscape=monitoring

I found this PNG but need a high-res format: https://static.distribooted.com/logos/netdata.png

You provide a URL or upload a zipped version of the logo here.

**Comment by ktsaou** [comment]
hm... this is not custom made.
It is this one: https://www.iconsdb.com/soylent-red-icons/seo-performance-icon.html

I you need a higher resolution than what is available there, I would have to hire someone to make it.

btw, most probably we will need to re-brand netdata. It seems the name "netdata" is too common and would probably hit some trademark. So, we may have change the name and the logo.

**Comment by ktsaou** [comment]
Thank you!

It does not appear on the list:
https://landscape.cncf.io/grouping=landscape&landscape=monitoring

Is this normal?

**Comment by ktsaou** [comment]
sorry! it appears! thank you!

**Comment by dankohn** [comment]
Also here: https://landscape.cncf.io/grouping=no&license=open-source&sort=stars

😉

**Comment by ktsaou** [comment]
> Also here: https://landscape.cncf.io/grouping=no&license=open-source&sort=stars

Wow! 4th by stars! Wow!

**Comment by ktsaou** [comment]
@dankohn can we use this logo in netdata? Can we merge it in the repo?

**Comment by dankohn** [comment]
Yes, please do. I hereby contribute the logos to you under the license of
your project.

--
Dan Kohn <dan@linuxfoundation.org>
Executive Director, Cloud Native Computing Foundation https://www.cncf.io
+1-415-233-1000 https://www.dankohn.com
